src/main/java/com/softserve/teamproject/entity/Location.java

```diff
@@ -28,19 +28,21 @@
   private String name;
 
   @ManyToOne
+  @JoinColumn(name = "coordinator_id", referencedColumnName = "id")
+  private User coordinator;
+
+  @ManyToOne
   @JoinColumn(name = "country_id", referencedColumnName = "id", nullable = false)
   private Country country;
 
-  @OneToOne
-  @JoinColumn(name = "coordinator_id", referencedColumnName = "id", nullable = false)
-  private User coordinator;
```

Comment:
I'd prefer not to change it right now. Please check if One-To-One works, and minimize changes if that schema works.